src/app/app-routing.module.ts

```diff
@@ -36,7 +36,7 @@ const routes: Routes = [
 ];
 
 @NgModule({
-  imports: [RouterModule.forRoot(routes)],
+  imports: [RouterModule.forRoot(routes, { relativeLinkResolution: 'legacy' })],
```

Comment:
Which parts of the app need this legacy relative link resolution?

Reply:
That was added by the auto-migrater, but it doesn't look like we need it; I'll go ahead and remove it